packages/transport-queue-dynamodb/package.json

```
@@ -1,19 +1,19 @@
 {
-  "name": "@credo-ts/didcomm-msg-pickup-repository-dynamodb",
+  "name": "@credo-ts/transport-queue-dynamodb",
```

```suggestion
  "name": "@credo-ts/didcomm-transport-queue-dynamodb",
```

Since we are now prefixing everything related to DIDComm in Credo 0.6.